contracts/StakingRewardsV2.sol

```diff
@@ -81,9 +75,6 @@ contract StakingRewardsV2 is
     /// @notice summation of rewardRate divided by total staked tokens
     uint256 public rewardPerTokenStored;
 
-    /// @inheritdoc IStakingRewardsV2
-    uint256 public cooldownPeriod;
-
     /// @notice represents the rewardPerToken
     /// value the last time the staker calculated earned() rewards
     mapping(address => uint256) public userRewardPerTokenPaid;
```

Comment:
it wont let me comment a couple lines below for some reason but ``mapping(address => uint256) public userLastStakeTime;`` should be removed because we remove the use of it everywhere in the code